src/math/Euler.js

```diff
@@ -280,10 +280,20 @@ THREE.Euler.prototype = {
 
 	},
 
-	toArray: function () {
+	toArray: function ( array, offset, withoutOrder ) {
 
-		return [ this._x, this._y, this._z, this._order ];
+		if ( array === undefined ) array = [];
+		if ( offset === undefined ) offset = 0;
 
+		array[ offset ] = this._x;
+		array[ offset + 1 ] = this._y;
+		array[ offset + 2 ] = this._z;
+
+		if( !withoutOrder ) {
```

Comment:
Hmmm... This can create confusion... Could we just keep it to `array` and `offset` and always save `this._order`? That way all these functions look the same.